src/documentation/pages/components/forms/file-input/examples/_sizes.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <ComponentExample title="Sizes" id="sizes" :tabs="exampleTabs">
+    <p class="-text" slot="example-description">
```

Comment:
As in the rest of the examples here we use `pug`.

```suggestion
<template lang="pug">
  <ComponentExample title="Sizes" id="sizes" :tabs="exampleTabs">
    p.-text(slot="example-description")
```